### PR TITLE
feat(external table): external table support sensitive column

### PR DESCRIPTION
**Pull request by zijiacj**
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/DBSchemaAccessor.java [hunk 1]
```diff
@@ -218,6 +218,16 @@ default List<String> showExternalTables(String schemaName) {
      */
     List<DBTableColumn> listBasicViewColumns(String schemaName, String viewName);
 
+    /**
+     * Get all external table columns(hold only basic info) in the specified schema
+     */
+    Map<String, List<DBTableColumn>> listBasicExternalTableColumns(String schemaName);
```
Comment (LuckyPickleZZ) [inline libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/DBSchemaAccessor.java:224]: I think `listExternalTableColumns` is enough.

File: server/odc-service/src/main/java/com/oceanbase/odc/service/datasecurity/SensitiveColumnService.java [hunk 2]
```diff
@@ -145,6 +147,12 @@ public List<DatabaseWithAllColumns> listColumns(@NotNull Long projectId, @NotEmp
                         accessor.listBasicTableColumns(database.getName()), exists));
                 databaseColumn.setView2Columns(getFilteringExistColumns(database.getId(),
                         accessor.listBasicViewColumns(database.getName()), exists));
+                DialectType dialectType = session.getDialectType();
```
Comment (LuckyPickleZZ) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/datasecurity/SensitiveColumnService.java:150]: Have you tested whether the external table can recognize sensitive columns properly?

File: libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/DBSchemaAccessor.java (hunk 1 above)
Comment (zijiacj) [reply libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/DBSchemaAccessor.java:224]: image
I recommend keeping the current method name the same as table and view. Only a small portion of the column object's field information is retrieved.
In fact, the implementation of the above three methods is to share the same sql statement, but now it is drawn into three methods and repeated writing three of the same sql, is it necessary to make an interface method?

File: server/odc-service/src/main/java/com/oceanbase/odc/service/datasecurity/SensitiveColumnService.java (hunk 2 above)
Comment (zijiacj) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/datasecurity/SensitiveColumnService.java:150]: image
image
yes

**Review by LuckyPickleZZ — DISMISSED** [review]

**Review by yhilmare — APPROVED** [review]
lgtm

File: libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/constant/StatementsFiles.java [hunk 3]
```diff
@@ -35,6 +35,8 @@ public final class StatementsFiles {
 
     public static final String MYSQL_5_6_x = "schema/sql/mysql/mysql_5_6_x.yaml";
 
+    public static final String OBORACLE_4_3_2 = "schema/sql/oboracle/oboracle_4_3_2.yaml";
```
Comment (yhilmare) [inline libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/constant/StatementsFiles.java:38]: prefer using 4.3.x, same as obmysql ode

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**